website/docs/connect-data/reference/appsmith-ai.md

```diff
@@ -0,0 +1,278 @@
+# Appsmith AI
+
+This page provides information on creating queries with Appsmith AI, which allows you to configure applications with advanced AI features such as text generation, image classification, and sentiment analysis without the need for any API keys or datasource authentication.
+
+
+
+
+<ZoomImage
+  src="/img/appsmith-ai-2.png" 
+  alt="Appsmith AI"
+  caption="Appsmith AI query"
+/>
+
+
+## Query Appsmith AI
+
+The following section is a reference guide that provides a description of the available commands with their parameters to create Appsmith AI queries.
+
+ You can try the examples provided in the query editor to understand and experiment with the capabilities of Appsmith AI.
+
+### Generate text 
+
+
+
+The Generate Text action allows you to generate text based on specific prompts.
+
+
+<dd>
+
+#### Prompt
+
+
+
+This field allows you to input prompts for text generation. You can also add data from queries or JSObjects using mustache bindings `{{}}`.
```

Comment:
Maybe add an example for a binding along with a hyperlink to the documentation page that talks about binding?